### DISCUSSION TITLE
Problem using workflow_dispatch trigger with other workflow triggers

### DISCUSSION THREAD

**Discussion by igorgiumellizup** [community/community#26780] [Actions]
<p>Hello</p>
<p>I’m getting the following problem when adding multiple workflow triggers:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/ZupIT/beagle/actions/runs/900466197/workflow" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/73198010/181099873-7fe5b27c-c4bd-4c09-a336-0a8c52a05a86.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_10x10.png"></div>

<h3><a href="https://github.com/ZupIT/beagle/actions/runs/900466197/workflow" target="_blank" rel="noopener nofollow ugc">ZupIT/beagle</a></h3>

<p><a href="https://github.com/ZupIT/beagle/actions/runs/900466197/workflow" target="_blank" rel="noopener nofollow ugc">ns/runs/900466197/workflow</a></p>

  <p><span class="label1">A framework to help implement Server-Driven UI in your apps natively.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>and</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/ZupIT/beagle/actions/runs/900466173/workflow" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/73198010/181099874-a0dc69e5-2110-45ad-9949-d55dfbdc40d4.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/6/466a266205bce54c91a9de0330393fc3ca6e6085_2_10x10.png"></div>

<h3><a href="https://github.com/ZupIT/beagle/actions/runs/900466173/workflow" target="_blank" rel="noopener nofollow ugc">ZupIT/beagle</a></h3>

<p><a href="https://github.com/ZupIT/beagle/actions/runs/900466173/workflow" target="_blank" rel="noopener nofollow ugc">ns/runs/900466173/workflow</a></p>

  <p><span class="label1">A framework to help implement Server-Driven UI in your apps natively.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Can’t we have a workflow that triggers with workflow_dispatch (manually) and other triggers?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can have multiple triggers, there’s just a syntax error in your workflow: There’s no colon after <code>workflow_dispatch</code>, making the value for the <code>on:</code> keyword a weird mix of string and object. Try this:</p>
<pre><code class="lang-yaml">on:
  workflow_dispatch:
  push:
    branches:
      - master
# ...
</code></pre>

**Comment by ferdnyc** [comment]
<p>Thanks for that, I don’t think I would’ve ever figured it out. I was close at one point, with attempts like this:</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - master
  workflow_dispatch:
jobs:
  # ...
</code></pre>
<p>But as I discovered, that doesn’t work.</p>
